experiments/functional/backup_and_restore/backup-restore.yml

```diff
@@ -5,17 +5,25 @@
 
       - name: Creating Backup
```

Comment:
@shashank855  can you please change the task name like `creating backup for cStor in minio backet' something like this